CHANGELOG.md

```
@@ -1,3 +1,11 @@
+## v13.4.1
+June 25 2026
+
+#### Fixed
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)